extensions/positron-python/python_files/posit/pinned-test-requirements.txt

```diff
@@ -7,7 +7,6 @@ hvplot==0.10.0 ; python_version >= '3.9'
 hvplot==0.8.0 ; python_version < '3.9'
 # TODO: 3.13 maint. install ibis when available for 3.13
 ibis-framework[duckdb]==9.5.0; python_version >= '3.10' and python_version < '3.13'
-ipykernel==6.29.5
```

Comment:
I removed these during dev. I think we'll probably still keep it in though, not 100% sure.

Reply:
Added it back.